Makefile

```diff
@@ -389,22 +389,28 @@ build-tools: goversion bin/buildhelper bin/envsubst bin/lsplatform
 build-tools-all: goversion bin/buildhelper bin/envsubst bin/lsplatform bin/catkubeletconfmap bin/watchnrtattr bin/mkginkgolabelfilter
 
 bin/buildhelper:
-	@go build -o bin/buildhelper tools/buildhelper/buildhelper.go
+	LDFLAGS="-static"
+	CGO_ENABLED=0 go build -o bin/buildhelper tools/buildhelper/buildhelper.go
 
 bin/envsubst:
-	@go build -o bin/envsubst tools/envsubst/envsubst.go
+	LDFLAGS="-static"
+	CGO_ENABLED=0 go build -o bin/envsubst tools/envsubst/envsubst.go
 
 bin/lsplatform:
-	@go build -o bin/lsplatform tools/lsplatform/lsplatform.go
+	LDFLAGS="-static"
+	CGO_ENABLED=0 go build -o bin/lsplatform tools/lsplatform/lsplatform.go
 
 bin/catkubeletconfmap:
-	@go build -o bin/catkubeletconfmap tools/catkubeletconfmap/catkubeletconfmap.go
+	LDFLAGS="-static"
+	CGO_ENABLED=0 go build -o bin/catkubeletconfmap tools/catkubeletconfmap/catkubeletconfmap.go
```

Comment:
this may be useful outside the build process, so OK to release. Ditto for `LDFLAGS`.